src/map/status.cpp

```diff
@@ -4864,6 +4870,28 @@ int status_calc_pc_sub(map_session_data* sd, uint8 opt)
 		}
 		if (sc->getSCE(SC_PORK_RIB_STEW))
 			sd->dsprate -= 2;
+		if( sc->getSCE(SC_TALISMAN_OF_FIVE_ELEMENTS) ) {
+			const std::vector<e_element> elements = { ELE_FIRE, ELE_WATER, ELE_WIND, ELE_EARTH, ELE_NEUTRAL };
+			int bonus = sc->getSCE(SC_TALISMAN_OF_FIVE_ELEMENTS)->val2;
+
+			for( const auto &element : elements ){
+				sd->indexed_bonus.magic_atk_ele[element] += bonus;
```

Comment:
https://www.divine-pride.net/database/skill/5423
```
Attaches the talisman of five elements to self or 1 party member to increases physical and magical damage against water, wind, earth, fire and neutral property enemies.
```
For me it should be
```suggestion
				sd->indexed_bonus.magic_addele[element] += bonus;
```
`magic_addele`: +x% magical damage against unit element e
while `magic_atk_ele` increases damage of e element magic of caster by x%